generators/app/templates/express-api/Dockerfile

```diff
@@ -1,11 +1,12 @@
-FROM alpine:3.3
```

Comment:
some people thing its better practice, I dont think its a big deal. But if it makes openshift work ;)